### PR TITLE
Add Exercise07

### PR DESCRIPTION
**Pull request by huynhngocminh2511**

### CODE REVIEW

File: Exercise07/Program.cs [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+            using (var strWriter = new StringWriter())
+            {
+                var writer = XmlWriter.Create(strWriter);
+                xmlSerializer.Serialize(writer, obj);
+                xmlString = strWriter.ToString();
+            }
+
+            Console.WriteLine(xmlString);
+
+            var xmldoc = new XmlDocument();
```
Comment (voquanghoa) [inline Exercise07/Program.cs]: Extract method

File: Exercise07/Program.cs [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+            Console.ReadKey();
+        }
+
+        static void SaveBusServiceIdtoXml(string id, BusServiceId busServiceId)
+        {
+            var xmlString = "";
+
+            using (var strWriter = new StringWriter())
+            {
+                var xmlSerializer = new XmlSerializer(busServiceId.GetType());
```
Comment (voquanghoa) [inline Exercise07/Program.cs]: Extract method

File: Exercise07/Program.cs [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+                var writer = XmlWriter.Create(strWriter);
+                xmlSerializer.Serialize(writer, busServiceId);
```
Comment (voquanghoa) [inline Exercise07/Program.cs]: Extract method

File: Exercise07/Program.cs [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+
+            var respone = service.GetBusService();
+            Console.WriteLine(respone.ResponseStatus);
+
+            var obj = respone.Data;
+            var xmlSerializer = new XmlSerializer(obj.GetType());
+
+            var xmlString = "";
+
+            using (var strWriter = new StringWriter())
```
Comment (voquanghoa) [inline Exercise07/Program.cs]: Extract method

File: Exercise07/Program.cs (hunk 1 above)
Comment (huynhngocminh2511) [reply Exercise07/Program.cs]: Done

File: Exercise07/Program.cs (hunk 2 above)
Comment (huynhngocminh2511) [reply Exercise07/Program.cs]: Done

File: Exercise07/Program.cs (hunk 3 above)
Comment (huynhngocminh2511) [reply Exercise07/Program.cs]: Done

File: Exercise07/Program.cs (hunk 4 above)
Comment (huynhngocminh2511) [reply Exercise07/Program.cs]: Done

**Pull request merged**